docs/current_docs/quickstart/conclusion.mdx

```diff
@@ -8,8 +8,10 @@ title: "Conclusion"
 
 ## Conclusion
 
-This quickstart has walked you through the process of Daggerizing an example application. It has taught you key Dagger concepts and introduced you to the Dagger API, the Dagger CLI, Dagger Functions and the Daggerverse.
+This Quickstart has walked you through the process of Daggerizing an example application. It has taught you key Dagger concepts and introduced you to the Dagger API, the Dagger CLI, Dagger Functions, and the Daggerverse.
 
-Now you have the tools to successfully take the next step: [adopting Dagger in your project](../adopting).
+Lastly, you should now have a [Dagger Cloud](https://dagger.cloud/) account, which helps you visualize your pipelines both pre-push and post-push in your CI. This will help you quickly troubleshoot and understand what's happening within any Daggerized pipeline. 
```

Comment:
It feels weird to me to specifically call out only Cloud like this for two reasons: Cloud is optional in this quickstart, and we don't have a similar callout for the other elements used in the quickstart. The segment `in your CI` is also inaccurate, we don't cover using Dagger Cloud with CI anywhere in the quickstart as the pipeline only runs locally. I would recommend removing this sentence.

Reply:
I saw this sentence still exists in original form and didn't see any response to my comment above. Unresolving as not sure if you missed this comment @mircubed ?

Reply:
sorry, maybe I updated the PR wrong? I made changes based on your comment here https://github.com/dagger/dagger/pull/8041/commits/136adb235c87a4dbe48978a2ae47ba243bdb523c#r1702323619